tests/phenotype/test_phenotype_resources.py

```diff
@@ -0,0 +1,395 @@
+        }
+        # Send post request
+        response = self.client.post(url_for(PHENOTYPES_LIST_URL),
+                                    headers=self._api_headers(),
+                                    data=json.dumps(kwargs))
+
+        # Check status code
+        self.assertEqual(response.status_code, 400)
+        # Check response body
+        response = json.loads(response.data.decode("utf-8"))
+        # Check error message
+        message = 'could not create phenotype'
+        self.assertIn(message, response['_status']['message'])
+        # Check field values
+        ph = Phenotype.query.first()
+        self.assertIs(ph, None)
+
+    def test_post_invalid_age(self):
+        """
+        Test create phenotype with bad input data
+
+        Participant with participant_id does not exist in db
+        """
+        # Create phenotype data
+        kwargs = {
+            'phenotype': 'Hand Tremor',
+            'hpo_id': 'HP:0002378',
+            'observed': 'positive',
+            # should be a positive integer
+            'age_at_event_days': -5,
+        }
+        # Send post request
+        response = self.client.post(url_for(PHENOTYPES_LIST_URL),
+                                    headers=self._api_headers(),
+                                    data=json.dumps(kwargs))
+
+        # Check status code
+        self.assertEqual(response.status_code, 400)
+
+        # Check response body
+        response = json.loads(response.data.decode("utf-8"))
+        # Check error message
+        message = 'could not create phenotype'
+        self.assertIn(message, response['_status']['message'])
+        # Check field values
+        ph = Phenotype.query.first()
+        self.assertIs(ph, None)
+
+    def test_post_bad_input(self):
+        """
+        Test create phenotype with bad input data
+
+        Participant with participant_id does not exist in db
+        """
+        # Create phenotype data
+        kwargs = {
+            'phenotype': 'Hand Tremor',
+            'age_at_event_days': 365,
+            'hpo_id': 'HP:0002378',
+            'observed': 'positive',
+            # kf_id does not exist
+            'participant_id': id_service.kf_id_generator('PT')()
+        }
+        # Send post request
+        response = self.client.post(url_for(PHENOTYPES_LIST_URL),
+                                    headers=self._api_headers(),
+                                    data=json.dumps(kwargs))
+
+        # Check status code
+        self.assertEqual(response.status_code, 400)
+
+        # Check response body
+        response = json.loads(response.data.decode("utf-8"))
+        # Check error message
+        message = '"{}" does not exist'.format(kwargs['participant_id'])
+        self.assertIn(message, response['_status']['message'])
+        # Check field values
+        ph = Phenotype.query.first()
+        self.assertIs(ph, None)
+
+    def test_post_multiple(self):
```

Comment:
Could be more concise by calling `_create_save_to_db()` twice.

Reply:
using different parameters for creating the second phenotype